README.md

```diff
@@ -1,285 +1,75 @@
+##  clone this repo at :
 
-<!--
-Example commands:
+```
+https://github.com/businesskaniki/budget.git
 
-```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
 ```
---->
+```
+cd budget
 
-### Install
+```
 
-Install this project with:
+### Install depedencies
 
-<!--
-Example command:
+```
+bundle install
 
-```sh
-  cd my-project
-  gem install
 ```
---->
 
 ### Usage
 
-To run the project, execute the following command:
-
-<!--
-Example command:
-
-```sh
-  rails server
 ```
---->
+rails s
 
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
-```sh
-  bin/rails test test/models/article_test.rb
 ```
---->
-
-### Deployment
-
-You can deploy this project using:
-
-<!--
-Example:
-
-```sh
+### test
 
 ```
- -->
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- AUTHORS -->
-
-## 👥 Authors <a name="authors"></a>
-
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FUTURE FEATURES -->
-
-## 🔭 Future Features <a name="future-features"></a>
-
-> Describe 1 - 3 features you will add to the project.
+bundle exec rspec
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+```
+## Authors
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+👤 **Nicholas Maina**
 
-<!-- CONTRIBUTING -->
+- GitHub: [@githubhandle](https://github.com/businesskaniki)
+- Twitter: [@twitterhandle](https://twitter.com/Nik7346)
 
-## 🤝 Contributing <a name="contributing"></a>
+## 🤝 Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- SUPPORT -->
-
-## ⭐️ Show your support <a name="support"></a>
-
-> Write a message to encourage readers to support your project
-
-If you like this project...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- ACKNOWLEDGEMENTS -->
-
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-> Give credit to everyone who inspired your codebase.
+Feel free to check the [issues page](https://github.com/businesskaniki/budget/issues).
 
-I would like to thank...
+## Show your support
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+Give us stars and share it with your friends ⭐️ if you like this project!
 
-<!-- FAQ (optional) -->
+## Acknowledgments
 
-## ❓ FAQ <a name="faq"></a>
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
Kindly make sure that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).